docs/queries/filters.md

```diff
@@ -43,22 +43,18 @@ When the day changes, relative dates like `due today` are re-evaluated so that t
 
 ## Matching - Workaround for Boolean OR
 
-All filters of a query have to match in order for a task to be listed.
-This means you cannot show tasks that have "GitHub in the path and have no due date or are due after 2021-04-04".
-Instead you would have two queries, one for each condition:
-
-    ### Not due
+Each line of a query has to match in order for a task to be listed.
```

Comment:
1. Trivia. The heading is wrong on line 46 now. What was a workaround for a missing feature is now actually implemented...

2. I think that there is so much to explain to users about how to use this that it will probably end up being a whole new page in its own right, so not worth spending too much time on the wording here.

3. But one thing I noted is that `NOT` on its own is also working, so should be added to the list, for example this works:

```typescript
    it('NOT', () => {
        // Arrange
        const filter = new BooleanField().createFilterOrErrorMessage(
            'NOT (description includes d1)',
        );

        // Act, Assert
        testWithDescription(filter, 'nothing', true);
        testWithDescription(filter, 'd1', false);
    });
```

Reply:
Added `NOT` to that list + added 2 tests centered around `NOT` on its own.

Reply:
Yes, that last bit was kind of my thinking. Also, the AND NOT and OR NOT are easier to understand if you know how NOT in isolation works.

Reply:
Yes, that last bit was kind of my thinking. Also, the AND NOT and OR NOT are easier to understand if you know how NOT in isolation works.